open-banking-brasil-certificate-standards-1_ID1.md

```diff
@@ -222,6 +222,12 @@ The following certifying authorities carried out the onboard process for Open Ba
 
 Front-End certificates are used to provide services, in general web pages, using TLS, which are accessed by the end user. Given their purpose, and to ensure greater interoperability, certificates must be of the EV (Extended Validation) type and must be generated through a valid certificte authority, following the rules defined in RFC 5280 and RFC 2818, in accordance with the principles and WebTrust criteria.
 
+### About certificates for exchanging information between Open Banking participating institutions and partners (non-participating)
```

Comment:
@RalphBragg this was included (at first just to the portuguese version) to reflect a requirement presented at BCB Regulation (Section IV of Resolução Conjunta nº 01 - https://www.in.gov.br/en/web/dou/-/resolucao-conjunta-n-1-de-4-de-maio-de-2020-255165055). Was it understandable :-)? I appreciate your suggestions to make it better :-)

Reply:
It makes sense @ginglass may i suggest

x509 Certificates that are used to secure communication channels between end-users browsers and banks services must be trusted be issued by a certificate authority certified by the CA Browser Forum. In addition,  and to ensure greater interoperability, certificates must meet the requirements for CAB Forum EV (Extended Validation), following the rules defined in RFC 5280 and RFC 2818, in accordance with the principles and WebTrust criteria.

or something like that